src/applications/disability-benefits/686/config/form.js

```diff
@@ -305,6 +311,10 @@ function createAddressUISchemaForKey(key, isRequiredCallback = () => true) {
       'ui:title': 'Post Office',
       'ui:required': (formData, index) =>
         isMilitaryAddress(get(`${insertRealIndexInKey(key, index)}`, formData)),
+      'ui:errorMessages': {
+        pattern: 'Please enter a valid post office',
+        required: 'Please enter a post officeAs ',
```

Comment:
It looks like there's an extra `As ` string at the end of this required message

Reply:
Oh, thanks! My bluetooth keyboard is wonky sometimes.